### ISSUE TITLE
Add label to search component

### ISSUE THREAD

**Issue by mvogelgesang** [wenzhixin/bootstrap-table#6568]
### Description

Website accessibility checkers point at the lack of an associated label tag or title attribute as an accessibility issue for the search component. This can most readily be seen using https://live.bootstrap-table.com/example/options/table-search.html and inspecting the search component that appears just above the table.  

This is somewhat related to #557 where the request was to add a visible label. I propose that the label be hidden by default and available for screen readers. 

I'd be happy to pickup this task and submit a PR
